lib/exercise_cases.rb

```diff
@@ -1,2 +1,30 @@
 require 'ostruct'
 require 'json'
+
+class ExerciseCase < OpenStruct
+  def name
+    'test_%s' % description.gsub(/[^\w ?!]/, '').gsub(/[- ]/, '_')
```

Comment:
This change causes the test names to change by not presenting hyphenated words as individual words, but rather as shown in this diff:

```plain
-  def test_large_distance_in_off_by_one_strand
+  def test_large_distance_in_offbyone_strand
```

I would like to place the change back in, due to things like "non_unique" being acceptable and readable, even if it technically isn't a separate word, and to avoid "off-by-one" becoming "offbyone".